syntax = "proto3";

package rpc_package;

// define a service
service CallInvoker {
  // define the interface and data type
  rpc call (CallRequest) returns (CallResponse) {}
}

// define the data type of request
message CallRequest {
  string route = 1;
  string params = 2;
  string description = 3;
}

// define the data type of response
message CallResponse {
  string status = 1;
  string value = 2;
  string message = 3;
}